cmd/autoheal/alerts_worker.go

```diff
@@ -183,28 +201,38 @@ func (h *Healer) runRule(rule *monitoring.HealingRule, alert *alertmanager.Alert
 	if err != nil {
 		return err
 	}
+	err = template.Process(action, alert)
+	if err != nil {
+		return err
+	}
 
-	// Decide which kind of action to run, and run it:
-	if rule.AWXJob != nil {
-		action := rule.AWXJob.DeepCopy()
-		err = template.Process(action, alert)
-		if err != nil {
-			return err
-		}
-		return h.runAWXJob(rule, action, alert)
-	} else if rule.BatchJob != nil {
-		action := rule.BatchJob.DeepCopy()
-		err = template.Process(action, alert)
-		if err != nil {
-			return err
+	if !h.throttling_disabled {
```

Comment:
If we get the same effect setting throttling to 0, then I think this isn't needed. If you still think that this is important, then consider adding a specific flag to enable/disable it:

```yaml
throttling:
  enabled: false
  interval: 1h
```

That way the user can temporarily disable it without having to modify the value of the `interval` parameter.

Also, please use camel case for variables: `throttlingDisabled`.